nautobot_golden_config/models.py

```diff
@@ -35,6 +36,19 @@
 VALIDATION_MSG = (
     ERROR_MSG + "Specifically the key {} was expected to be of type(s) {} and the value of {} was not that type(s)."
 )
+PLATFORM_LOOKUP_TABLE = {
+    "cisco_ios": "ios",
+    "cisco_xe": "iosxe",
+    "cisco_xr": "iosxr",
+    "cisco_nxos": "nxos",
+    "arista_eos": "eos",
+}
```

Comment:
Should this go to netutils library?

Reply:
Yes: https://github.com/networktocode/netutils/issues/233